ansible/ansible_playbook.yml

```
@@ -0,0 +1,69 @@
+- name: Run To-Do App
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: What is the Trello API key?
+    - name: trello_api_token
+      prompt: What is the Trello API token?
+    - name: trello_board_id
+      prompt: What is the Trello board id?
+    - name: trello_todo_list_id
+      prompt: What is the Trello todo list id?
+    - name: trello_doing_list_id
+      prompt: What is the Trello doing list id?
+    - name: trello_done_list_id
+      prompt: What is the Trello done list id?
+
+  tasks:
+    - name: Install Git and Python3
+      ansible.builtin.yum:
+        name: "{{ packages }}"
+      vars:
+        packages:
+          - git
+          - python311
```

Nothing wrong with this and definitely just preference, but I've seen it a few times and I almost always prefer just putting the packages directly under name!
```suggestion
      ansible.builtin.yum:
        name:
          - git
          - python311
```